tools/server/webui/svelte.config.js

```diff
@@ -8,6 +8,11 @@ const config = {
 	// for more information about preprocessors
 	preprocess: [vitePreprocess(), mdsvex()],
 	kit: {
+		paths: {
+			// Base path for deployment in subdirectories (e.g., '/llama' for hosting at example.com/llama)
+			// Leave empty ('') for root deployment (default behavior)
+			base: ''
```

Comment:
It'd be best to allow defining the base path with an env var, e. g.:

```suggestion
			base: process.env.BASE_PATH || ''
```

So that we can run

```bash
BASE_PATH="/custom-path" npm run build
```